src/app/app.module.ts

```diff
@@ -36,7 +36,7 @@ import { FeaturesComponent } from './components/features/features.component';
 
 import { Backend } from './services/backend';
 import { MaterialIconsService } from './services/material-icons/material-icons.service';
-import { Recipe} from './models/recipe.model';
+import { Recipe } from './models/recipe.model';
```

Comment:
yep. noticed this also and was going to mention it at some point.I do have generic rules in vs code for this. I think the spaces are the standard right. makes sense!